class-toc-plus.php

```diff
@@ -325,23 +325,45 @@ public function shortcode_sitemap( $atts ) {
 
 
 		public function shortcode_sitemap_pages( $attributes ) {
+			global $post;
+			$args_child_of = 0;
 			$atts = shortcode_atts(
 				[
 					'heading'      => $this->options['sitemap_heading_type'],
 					'label'        => $this->options['sitemap_pages'],
 					'no_label'     => false,
 					'exclude'      => '',
 					'exclude_tree' => '',
+					'child_of'     => $this->options['child_of']
```

Comment:
`$this->options['child_of']` doesn't exist in the class's options property, so this can be

`'child_of' => 0`